src/Microsoft.DotNet.Build.CloudTestTasks/UploadToAzure.cs

```diff
@@ -1,96 +1,229 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.IO;
-using Microsoft.Build.Framework;
-using Microsoft.Build.Utilities;
-using Microsoft.WindowsAzure.Storage;
-using Microsoft.WindowsAzure.Storage.Blob;
+// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="UploadToAzure.cs" company="">
+//   
+// </copyright>
+// <summary>
+//   The upload to azure.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
 
 namespace Microsoft.DotNet.Build.CloudTestTasks
 {
-    public sealed class UploadToAzure : Task
+    #region
+
+    using System;
+    using System.Collections.Generic;
+    using System.Globalization;
+    using System.IO;
+    using System.Net;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using System.Xml;
+
+    using Microsoft.Build.Framework;
+
+    using Task = Microsoft.Build.Utilities.Task;
+
+    #endregion
+
+    /// <summary>
+    ///     The upload to azure.
+    /// </summary>
+    public class UploadToAzure : Task, ICancelableTask
     {
         /// <summary>
-        /// The Azure account name used when creating the connection string.
+        /// The token source.
         /// </summary>
-        [Required]
-        public string AccountName { get; set; }
+        private CancellationTokenSource tokenSource;
+
+        /// <summary>
+        ///     Initializes a new instance of the <see cref="UploadToAzure" /> class.
+        /// </summary>
+        public UploadToAzure()
+        {
+            this.Overwrite = false;
```

Comment:
False is already the default for booleans (and if you wanted to be explicit anyway, you could just add "= false" to the end of the property line to avoid having to add the constructor)